contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -96,6 +97,7 @@ contract LoanTokenLogicStandard is LoanTokenLogicStorage {
     function burn(address receiver, uint256 burnAmount)
         external
         nonReentrant
+        globallyNonReentrant
```

Comment:
i think we should also add this to borrow and margin trade

Reply:
do i just not see it or are there indeed no global reentrancy checks on the closing functions of the protocol?

Reply:
Reentrancy checks has been added to closeWithSwap, closeWithDeposit, liquidate & rollover